### PR TITLE
Add actions

### PR DESCRIPTION
**Pull request by aheimsbakk**
Added CI

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      - '**.md'
+      - '.github/workflows/*yml'
+
+  # schedule full rebuild and push on schedule, see todos
+  schedule:
+    - cron:  '13 3 * * *'
+
+env:
+  # TODO: remember to update version on new tag
+  LATEST_TAG: 2.4.2
```
Comment (mpejcoch) [inline .github/workflows/docker.yml:31]: This looks like pycsw version. Did you also think about how to deal with the "package"/container versions? I.e., we might release 2.4.2 a couple of times before it's OK. Something like the release's versioning for distro packages: 2.4.2-1, 2.4.2-2...

File: .gitignore [hunk 2]
```diff
@@ -1 +1,3 @@
 *swp
+*log
```
Comment (mpejcoch) [inline .gitignore:2]: *.log, *.swp?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # pycsw
 
-This is MET norways base container for pycsw, an OGC CSW server implementation written in Python. For more information about pycsw got to [pycsw.org](https://pycsw.org). For the source code of pycsw got to [gepython/pycsw](https://github.com/geopython/pycsw) on GitHub.
+This is MET Norways base container for pycsw, an OGC CSW server implementation written in Python. For more information about pycsw got to [pycsw.org](https://pycsw.org). For the source code of pycsw got to [gepython/pycsw](https://github.com/geopython/pycsw) on GitHub.
```
Comment (mpejcoch) [inline README.md]: Norway's

**Review by mpejcoch — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (aheimsbakk) [reply .gitignore:2]: Vagrant leftover log. And my own vim .swp log, so I don't see it when I write git status.

File: .github/workflows/docker.yml (hunk 1 above)
Comment (aheimsbakk) [reply .github/workflows/docker.yml:31]: For this container I didn't consider that. I don't think we want to keep those versions around for posterity. As I see it dev should be used for those. Would it make any sense to have 

* `2.4.2-2`, `2.4`, `2`, `latest`
* `2.4.2-1`

I've not seen that done anywhere else with containers.

I think they fix the `2.4.2` version if they find a bug in the current container to ensure everyone get the correct version.

File: README.md (hunk 3 above)
Comment (aheimsbakk) [reply README.md]: Pushed changes.

File: .gitignore (hunk 2 above)
Comment (mpejcoch) [reply .gitignore:2]: I was thinking if *log is not too restrictive, it would "eat" a "catalog" too :)

**Review by mpejcoch — APPROVED** [review]

File: .github/workflows/docker.yml (hunk 1 above)
Comment (mpejcoch) [reply .github/workflows/docker.yml:31]: I've seen it with the distro packages, not containers. Not sure if it's worth the trouble.

**Pull request merged**